### ISSUE TITLE
Run VS Code debugger with --dap.Suspend=false

### ISSUE THREAD

**Issue by nirvdrum** [oracle/graal#5147]
**Feature request**

Please include the following information:

**Is your feature request related to a problem? Please describe.**

The default DAP options are to pause on the first line of execution in the debugger. While I appreciate that being useful in a CLI debugging session, it's not how most GUI debuggers operate. I keep starting a debug session and switching over to another task until my breakpoint gets hit, only to switch back and discover that the debugger is waiting on the first line. If that process were faster, maybe it wouldn't be so bad, but it takes a while to even reach the first line so it's easy to think the debugger is going to run until it hits the first declared breakpoint.

**Describe the solution you'd like.**

I'd like the VS Code debugger to only consider breakpoints I've declared. If none are set, the application should run to completion.

**Describe who do you think will benefit the most.**
GraalVM users. Truffle language users.

**Describe alternatives you've considered.**

I've tried to pass the `--dap.Suspend=false` option myself, but the launch.json config doesn't have a section for DAP options. Instead, it's an awkward process of trying to slot it in with the application configuration. I have yet to get it to work correctly. It's unclear if that's a debugger issue or if I'm supplying the arguments incorrectly.

**Comment by entlicher** [comment]
I don't feel really comfortable changing the default value of suspend, but the `suspend` option is now present in the launch configurations by default, at least. This allows the user to immediately notice the option and change it's value when desired.
https://github.com/graalvm/vscode-extensions/commit/b230ba5d6d42acf6efd65ed1c7bc5ad968fe74cb
